client/src/app/pages/license-list/license-table.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { generatePath, NavLink } from "react-router-dom";
+import { Table, Tbody, Td, Th, Thead, Tr } from "@patternfly/react-table";
+
+import { SimplePagination } from "@app/components/SimplePagination";
+import {
+  ConditionalTableBody,
+  TableHeaderContentWithControls,
+  TableRowContentWithControls,
+} from "@app/components/TableControls";
+import { Paths } from "@app/Routes";
+import { LicenseSearchContext } from "./license-context";
+import { WithSBOMsByLicense } from "@app/components/WithSBOMsByLicense";
+import { LoadingWrapper } from "@app/components/LoadingWrapper";
+import { Skeleton } from "@patternfly/react-core";
+import { TableCellError } from "@app/components/TableCellError";
+import { WithPackagesByLicense } from "@app/components/WithPackagesByLicense";
+
+export const LicenseTable: React.FC = () => {
+  const { isFetching, fetchError, totalItemCount, tableControls } =
+    React.useContext(LicenseSearchContext);
+
+  const {
+    numRenderedColumns,
+    currentPageItems,
+    propHelpers: {
+      paginationProps,
+      tableProps,
+      getThProps,
+      getTrProps,
+      getTdProps,
+    },
+  } = tableControls;
+
+  return (
+    <>
+      <Table {...tableProps} aria-label="License table">
+        <Thead>
+          <Tr>
+            <TableHeaderContentWithControls {...tableControls}>
+              <Th {...getThProps({ columnKey: "name" })} />
+              <Th {...getThProps({ columnKey: "packages" })} />
+              <Th {...getThProps({ columnKey: "sboms" })} />
+            </TableHeaderContentWithControls>
+          </Tr>
+        </Thead>
+        <ConditionalTableBody
+          isLoading={isFetching}
+          isError={!!fetchError}
+          isNoData={totalItemCount === 0}
+          numRenderedColumns={numRenderedColumns}
+        >
+          {currentPageItems.map((item, rowIndex) => {
+            return (
+              <Tbody key={item.license}>
+                <Tr {...getTrProps({ item })}>
+                  <TableRowContentWithControls
+                    {...tableControls}
+                    item={item}
+                    rowIndex={rowIndex}
+                  >
+                    <Td
+                      width={35}
```

Comment:
Let's give the max width possible here. I tested it locally and this is how it looks 

<img width="1603" height="977" alt="Image" src="https://github.com/user-attachments/assets/0adbf554-91a4-47c1-ac14-54eeeef83ff5" />

The license columns width is too small for the license name.